channels-rke2.yaml

```diff
@@ -234,6 +240,9 @@ releases:
       <<: *agentArgs-v1-21-5-rke2r2
     charts: &charts-v1-22-4-rke2r1
       <<: *charts-v1-21-7-rke2r1
-      rke2-canal:
+      rke2-calico:
```

Comment:
do we not need both?

Reply:
We do, but we should get canal for free from the 1.21 bit since the versions are the same
https://github.com/rancher/rke2/blob/release-1.21/Dockerfile#L97 vs https://github.com/rancher/rke2/blob/master/Dockerfile#L99

Reply:
rke2-canal is already defined in line 207, so we had rke2-canal twice? :thinking:

Reply:
Yeah we had it explicitly defined in the 1.22.4 section as well, but don't need it since we pull from 1.21.7 via:
```

Reply:
We're going to need some automation to either build or validate these chart and args sections. I have an issue assigned to me to do this, but this miss here points out why manual validation is problematic.